src/test/java/org/mockbukkit/mockbukkit/block/data/AgeableDataMockTest.java

```diff
@@ -100,4 +101,15 @@ void givenAge15Material()
 
 	}
 
+	@Test
+	void validateClone()
+	{
+		ageable.setAge(3);
+
+		@NotNull AgeableDataMock cloned = ageable.clone();
```

Comment:
This test doesn't really test anything?
I would do:
```
		AgeableDataMock cloned = ageable.clone();
		assertEquals(ageable, cloned);
		assertEquals(ageable.getAge(), cloned.getAge());
		ageable.setAge(3);
		assertNotEquals(ageable, cloned);
		assertNotEquals(ageable.getAge(), cloned.getAge());
```

Reply:
Thank you for the feedback, i'll review the tests and include a test to remind to implement the clone :)